rest/route/github.go

```diff
@@ -370,26 +373,45 @@ func (gh *githubHookApi) handleGitTag(ctx context.Context, event *github.PushEve
 	}
 	catcher := grip.NewBasicCatcher()
 	for _, pRef := range projectRefs {
-		// if a version for this revision exists for this project, add tag
-		existingVersion, err := gh.sc.FindVersionByProjectAndRevision(pRef.Id, hash)
-		if err != nil {
-			catcher.Add(errors.Wrapf(err, "problem finding version for project '%s' with revision '%s' to add tag '%s'",
-				pRef.Id, hash, tag.Tag))
-			continue
+		var existingVersion *model.Version
+		noExistingVersionMessage := message.Fields{
```

Comment:
Should cleanup debugging logging (it looks like this whole thing can be moved into 411)